src/block.rs

```diff
@@ -16,7 +16,7 @@ pub type BlockResult<T> = core::result::Result<T, BlockError>;
 
 /// Represent a certain amount of data from a block device.
 #[derive(Clone)]
-#[repr(C, align(512))]
+#[repr(C)]
```

Comment:
```suggestion
#[repr(C, align(2)] // PRDT entries in AHCI must be word aligned
```

Reply:
Is `storage_device` supposed to be tied to AHCI though? I thought it was meant to be an abstraction layer. IMO alignment should be handled on a higher layer.